### DISCUSSION TITLE
Assistance Required: Email Verification Issue with Cattr.app Community

### DISCUSSION THREAD

**Discussion by Shaerif** [cattr-app/server-application#280] [Q&A] [answered]
Cattr.app Support Team,
I am experiencing difficulty in verifying my email to join the Cattr.app Community. Despite multiple attempts to resend the confirmation email, I encounter errors stating "Oops! Something went wrong. Please reload the page and try again." or "You're going a little too quickly. Please try again in a few seconds." Could you please help me resolve this issue?

Thank you for your attention to this matter.

**Comment by DanilShapilov** [comment, marked as answer]
Hi, please use [github discussions](https://github.com/orgs/cattr-app/discussions) instead. Forum will be closed at some point.
